package.json

```diff
@@ -27,29 +30,40 @@
     "url": "https://github.com/netlify/netlify-lambda/issues"
   },
   "dependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/plugin-transform-object-assign": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "babel-loader": "^8.0.0",
-    "body-parser": "^1.18.3",
-    "commander": "^2.17.1",
-    "express": "^4.16.3",
+    "@babel/core": "^7.10.4",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.10.4",
+    "@babel/plugin-transform-object-assign": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
+    "babel-loader": "^8.1.0",
+    "body-parser": "^1.19.0",
+    "commander": "^5.1.0",
+    "express": "^4.17.1",
     "express-logging": "^1.1.1",
-    "globby": "^10.0.1",
+    "globby": "^11.0.1",
     "jwt-decode": "^2.2.0",
-    "toml": "^2.3.3",
-    "webpack": "^4.17.1",
-    "webpack-merge": "^4.1.4"
+    "toml": "^3.0.0",
+    "webpack": "^4.43.0",
+    "webpack-merge": "^5.0.9"
   },
   "devDependencies": {
-    "auto-changelog": "^1.13.0",
+    "@commitlint/cli": "^9.0.0",
+    "@commitlint/config-conventional": "^9.0.0",
+    "auto-changelog": "^2.2.0",
+    "eslint": "^7.0.0",
     "gh-release": "^3.5.0",
-    "jest": "^23.6.0",
+    "husky": "^4.0.7",
+    "jest": "^26.1.0",
+    "node-fetch": "^2.6.0",
+    "prettier": "^2.0.0",
     "rimraf": "^3.0.2"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0"
```

Comment:
Breaking change (we need node 10 to update to the latest dependencies).
Since we're planning on doing a major version this should be fine.